src/utils/database.ts

```diff
@@ -3,11 +3,7 @@ let dbClient: MongoClient;
 let eventDbClient: MongoClient;
 export async function initDbClient() {
   try {
-    dbClient = await MongoClient.connect(process.env.DATABASE_URI || "", {
-      useNewUrlParser: true,
-      useUnifiedTopology: true,
-      ignoreUndefined: true,
-    });
+    dbClient = await MongoClient.connect(process.env.DATABASE_URI || "");
```

Comment:
Throw an error and do lrocess.exit if db not connected or db string not found

Reply:
![image](https://github.com/srm-kzilla/jack/assets/69302420/ebcaafc3-a747-488d-b52d-58247ab43ee3)
It doesn't exit when I give process.exit()

Reply:
Completed